Whiskey/Functions/Install-WhiskeyTool.ps1

```diff
@@ -253,8 +253,22 @@ function Install-WhiskeyTool
 
                             if( -not (Test-Path -Path $nodePath -PathType Leaf) )
                             {
-                                Write-Verbose -Message ('{0} x {1} -o{2} -y' -f $7z,$nodeZipFile,$nodeRoot)
-                                & $7z 'x' $nodeZipFile ('-o{0}' -f $nodeRoot) '-y'
+                                #if( $IsWindows )

```

Comment:
Sadly you'll need to un-comment this code because it caused the build to fail, I guess we'll need to rely on 7-Zip for this.